mmv1/templates/terraform/examples/security_policy_rule_basic.tf.erb

```diff
@@ -0,0 +1,23 @@
+resource "google_compute_security_policy" "default" {
+  provider    = google-beta
```

Comment:
I'm still in the middle of reviewing this PR. A quick suggestion is removing `provider    = google-beta` in the examples as this resource + its examples are no longer marked as beta-only in the latest commit